src/gtfs/mod.rs

```diff
@@ -44,11 +46,20 @@ fn add_prefix_to_collections(prefix: String, collections: &mut Collections) -> R
     Ok(())
 }
 
-pub fn read<P: AsRef<path::Path>>(
-    path: P,
-    config_path: Option<P>,
-    prefix: Option<String>,
-) -> Result<Model> {
+/// Imports a `Model` from the [GTFS](http://gtfs.org/) files in the
+/// `path` directory.
+///
+/// The `config_path` argument allows you to give a file containing a
```

Comment:
It's not very clear whether `config_path` is a path to a directory that contains a Json, or a path to the Json file itself.